common/spec/dependabot/notices_spec.rb

```diff
@@ -35,9 +35,6 @@ def deprecated?
 
   sig { override.returns(T::Boolean) }
   def unsupported?
-    # Check if the feature flag for Bundler v1 unsupported error is enabled.
-    return false unless Dependabot::Experiments.enabled?(:bundler_v1_unsupported_error)
```

Comment:
let's make sure to remove the flag everywhere too :)

Reply:
Yes, actually I am removing from everywhere and not only unsupported but deprecation flag as well. We are done now checking last issues. After core is done, I will also remove them feature flags from `dependabot-api` as well. 

- `add_deprecation_warn_to_pr_message`
- `bundler_v1_unsupported_error`

Reply:
Here is the PR removing from dependabot-api as well: https://github.com/github/dependabot-api/pull/5816